build/build-viewer.js

```diff
@@ -52,11 +51,26 @@ async function main() {
     ],
     javascripts: [
       reportGeneratorJs,
+      'window.ReportGenerator = window.ReportGenerator.ReportGenerator',
```

Comment:
Can you add a TODO to remove this for https://github.com/GoogleChrome/lighthouse/pull/13429